src/bio/bio_config.c

```diff
@@ -960,10 +960,7 @@ bio_read_auto_faulty_criteria(const char *nvme_conf, bool *enable, uint32_t *max
 	if (rc != 0) {
 		if (rc == JSON_NOT_FOUND) {
 			rc             = 0;
```

Comment:
Sound strange to keep `rc == 0` in such situations.  It will be confusing for the caller to know that the value of `enable`, `max_io_errs` and `max_csum_errs` have not been set.

Reply:
So in this situation we have not found any custom configuration in the supplied JSON, this is a valid situation and in this case we want to continue and not block loading the rest of the config. max_*_errs will retain their default values and auto_faulty will stay enabled.

Reply:
After more investigation and validation with @tanabarr , there is no issue as the param value are previously  properly initialized.